src/browser.js

```diff
@@ -181,7 +181,7 @@ function formSubmitHandler(event, force = false) {
 
       $(el).trigger('submit', true);
     }).catch(function(err) {
-      console.error(err);
+      // console.error(err);
```

Comment:
Let's keep this `console.error` here since otherwise errors get swallowed by the promise and disappear.